test/test_public_bindings.py

```diff
@@ -279,8 +285,8 @@ def onerror(modname):
                 importlib.import_module(modname)
             except Exception as e:
                 # Some current failures are not ImportError
-
-                failures.append((modname, type(e)))
+                log.exception("import_module failed")
```

Comment:
What's the purpose of this message? Why should it be here?

Reply:
We're currently suppressing the backtrace of the exception here, and instead of figuring out how to render it nicely below, I'm just dumping into the logs so you can look for it.

Reply:
I guess one downside to this log is that you will also get all the whitelisted modules dumped to log lol